packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -1576,8 +1576,6 @@ export interface IBaseExtendedPickerState<T> {
     // (undocumented)
     queryString: string | null;
     // (undocumented)
-    selectedItems: T[] | null;

```

Comment:
Even though `selectedItems` isn't being used, this is still a breaking API change - should this go in a major release? @joschect what do you think?

Reply:
I think that this is fine. Components are supposed to not expose state directly so I don't think that this would technically break the api.

Reply:
Required revert in #9663